packages/iov-bns/src/decode.spec.ts

```diff
@@ -194,6 +195,46 @@ describe("Decode", () => {
     });
   });
 
+  describe("decodeCashConfiguration", () => {
+    it("can decode configuration with non-null minimal fee", () => {
+      const config: codecImpl.cash.Configuration = {
```

Comment:
Can you change the argument of `decodeCashConfiguration` to `cash.IConfiguration`? Then we can we can use data-only `cash.IConfiguration` here as well and don't need the dummy `toJSON`. Also `owner` and `collectorAddress` become optional and can be skipped for now